<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="daysTillColText">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Days till Moon collision</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="storyTextBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="torchOnBt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>On</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="torchOffBt">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>490</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Off</string>
    </property>
   </widget>
   <widget class="QLabel" name="torchText">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Torch:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>20</width>
      <height>551</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="minimap">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>420</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="viewMapBt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>View full map </string>
    </property>
   </widget>
   <widget class="QProgressBar" name="healthBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="hpText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Health Points</string>
    </property>
   </widget>
   <widget class="QListView" name="invListView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>91</width>
      <height>251</height>
     </rect>
    </property>
    <property name="viewMode">
     <enum>QListView::IconMode</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="entityView">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="northBt">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>420</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>North</string>
    </property>
   </widget>
   <widget class="QPushButton" name="southBt">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>South</string>
    </property>
   </widget>
   <widget class="QPushButton" name="southBt_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>West</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eastBt">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>East</string>
    </property>
   </widget>
   <widget class="QLabel" name="attackWithText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Attack with</string>
    </property>
   </widget>
   <widget class="QLabel" name="giveTradeText">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>310</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Give/Trade</string>
    </property>
   </widget>
   <widget class="QListView" name="attackListView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="giveTradeListView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>340</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
